components/pipeline-service/development/kustomization.yaml

```diff
@@ -10,7 +10,10 @@ commonAnnotations:
 resources:
   - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd/pipeline-service?ref=eb6e265ddc08256cac7a5c4b22cba663d9c7ef26
   - git::https://github.com/openshift-pipelines/pipeline-service.git/developer/openshift/gitops/argocd/pipeline-service-storage?ref=eb6e265ddc08256cac7a5c4b22cba663d9c7ef26
-
+  # - ../base/external-secrets
```

Comment:
just remove this line

Reply:
please refer to the above comment. This seems to be referring to the same file.